build.gradle

```diff
@@ -11,6 +11,13 @@ allprojects {
     tasks.withType(AbstractCopyTask) {
       duplicatesStrategy = DuplicatesStrategy.EXCLUDE
     }
+
+    repositories {
```

Comment:
I am curious why do we need that? All artifacts has to be pulled from official repos

Reply:
This was a failed attempt of mine to get my IDE to recognize the thirdparty shaded jars, will revert this change.